src/batch/src/executor2/hash_agg.rs

```diff
@@ -128,13 +129,13 @@ impl BoxedExecutorBuilder for HashAggExecutorBuilder {
 }
 
 /// `HashAggExecutor` implements the hash aggregate algorithm.
-pub(super) struct HashAggExecutor<K> {
+pub(crate) struct HashAggExecutor2<K> {
     /// factories to construct aggregator for each groups
     agg_factories: Vec<AggStateFactory>,
     /// Column indexes of keys that specify a group
     group_key_columns: Vec<usize>,
     /// child executor
-    child: BoxedExecutor,
+    child: BoxedExecutor2,
     /// hash map for each agg groups
     groups: AggHashMap<K>,
```

Comment:
We nolonger need these to store as fields.

Reply:
fixed